src/bindgen/ir/ty.rs

```diff
@@ -389,6 +389,7 @@ impl Type {
         generic.simplify_standard_types();
 
         match path.name() {
+            "Box" => Some(Type::Ptr(Box::new(generic))),
```

Comment:
I'm happy to add a `FIXME` comment and file an issue for unsized types, similar to below, if this is otherwise acceptable.

Reply:
Yeah, sounds sensible, but please add a comment pointing to the documentation.